modules/scip/metadata.json

```diff
@@ -11,6 +11,7 @@
     ],
     "versions": [
         "8.1.0"
```

Comment:
Nit: Missing comma in list for valid json

Reply:
fixed.